src/cast.c

```diff
@@ -100,8 +100,28 @@ Expression *implicitCastTo(Expression *e, Scope *sc, Type *t)
                 //printf("type %p ty %d deco %p\n", type, type->ty, type->deco);
                 //type = type->semantic(loc, sc);
                 //printf("type %s t %s\n", type->deco, t->deco);
-                e->error("cannot implicitly convert expression (%s) of type %s to %s",
-                    e->toChars(), e->type->toChars(), t->toChars());
+                char hint[128];
```

Comment:
There is no need to use a buffer to store string literals, just use a `const char*`